lib/page_ez/method_generators/has_many_ordered_selector.rb

```diff
@@ -26,27 +25,31 @@ def run(target)
         dynamic_options = @dynamic_options
         constructor = @base_selector.run(target)
 
-        evaluator_class = SelectorEvaluator.build(@name, dynamic_options: dynamic_options, options: options, selector: build_selector)
+        evaluator_class = SelectorEvaluator.build(@name, dynamic_options: dynamic_options, options: options, selector: selector)
 
         target.logged_define_method("#{singularized_name}_at") do |index, *args|
-          evaluator = evaluator_class.run([index].dup.push(*args), target: self)
+          evaluator = evaluator_class.run(args, target: self)
           HasOneResult.new(
             container: container,
-            selector: evaluator.selector,
+            selector: build_selector[evaluator.selector, index],
```

Comment:
Nice change, it's great to be able to keep the index out of the evaluator class arguments.